src/MasterBlocks.php

```diff
@@ -173,6 +173,7 @@ public function enqueue_block_public_assets(): void
     private function get_p4_options(): array
     {
         $option_values = get_option('planet4_options');
+        $is_new_ia = $option_values['new_ia'] === 'on' ? true : false;
```

Comment:
Isn't `$is_new_ia = $option_values['new_ia'] === 'on';` enough? It's already a boolean